pkg/metricscollector/opentelemetry.go

```diff
@@ -180,6 +182,15 @@ func initMeters() {
 		otLog.Error(err, msg)
 	}
 
+	_, err = meter.Float64Histogram(
+		"keda.internal.scale.loop.latency.bucket",
+		api.WithDescription("Internal latency of ScaledObject/ScaledJob loop execution"),
+		api.WithUnit("s"),
+	)
+	if err != nil {
+		otLog.Error(err, msg)
+	}
+
```

Comment:
Isn't this the same as the metric just above?

Reply:
Well, yes, but unlike the above one, this should be a histogram. But I am not that sure when it comes to otel.